Failing OSSLifecycle test

:clock11: **When did the problem start?**

Since yesterday.

:camera: **Live badge**

<!-- Provide a link to the live badge in plain text and markdown.  -->

![OSS Lifecycle](https://img.shields.io/osslifecycle/Teevity/ice)

:wrench: **Is the live badge working?**

Live badge working.

:link: **CircleCI link**

https://circleci.com/gh/badges/daily-tests/527

:beetle: **Stack trace**

```
AssertionError: message mismatch: expected 'not found' to equal 'active'
```

:bulb: **Possible solution**

The OSSMETADATA file was removed (see https://github.com/zalando/ghe-backup/pull/108), we should find a different example.

--- a/services/osslifecycle/osslifecycle.tester.js
+++ b/services/osslifecycle/osslifecycle.tester.js
@@ -8,7 +8,7 @@ const t = (module.exports = new ServiceTester({
 }))
 
 t.create('osslifecycle active status')
-  .get('/zalando/ghe-backup.json')
+  .get('/netflix/sureal.json')
   .expectBadge({
     label: 'oss lifecycle',
     message: 'active',

